### PR TITLE
Fix Owna protocol tvl by calling contract

### PR DESCRIPTION
**Pull request by 0xRoch**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/owna exports TVL: 
        
 
 
    polygon_zkevm             999.846
    
    total                    999.846

**Comment by llamatester** [comment]
The adapter at projects/owna exports TVL: 
        
 
 
    polygon_zkevm             999.846
    
    total                    999.846

File: projects/owna/index.js [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+    }
+
+    return {
+      usd: total / 10 ** 6,
+    };
+  }
+
   module.exports[chain] = {
-    tvl: sumTokensExport({ chain, tokens, owners: holders })
+    tvl: totalBorrowed
```
Comment (g1nt0ki) [inline projects/owna/index.js:41]: we can count the borrowed amount under 'borrowed' key but not as tvl https://docs.llama.fi/list-your-project/what-to-include-as-tvl

**Comment by llamatester** [comment]
The adapter at projects/owna exports TVL: 
        
 
 
    polygon_zkevm             50.082867
    polygon_zkevm-borrowed    1.00 k
    borrowed                  1.00 k
    
    total                    50.082867

**Comment by llamatester** [comment]
The adapter at projects/owna exports TVL: 
        
 
 
    polygon_zkevm             50.007943030968
    polygon_zkevm-borrowed    998.5039999999999
    borrowed                  998.5039999999999
    
    total                    50.007943030968

**Pull request merged**